crowbar_framework/app/assets/javascripts/application.js

```diff
@@ -256,4 +257,41 @@ jQuery(document).ready(function($) {
     },
     500
   );
+
+  // Backup and restore
+  $('[data-toggle="tooltip"]').tooltip({html : true});
+  $('[data-toggle="inline"]').popover({
+    html : true,
+    content: function() {
+      return $($(this).data('inline')).html();
+    }
+  });
+
+  $('[name="fileupload"]').fileinput({
```

Comment:
`[name="fileupload"]` is not a really generic selector. Maybe use it for fileupload form type?